client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -195,76 +203,113 @@ const PluginsBrowser = React.createClass( {
-					placeholder={ this.translate( 'Search Plugins' ) }
-					delaySearch={ true }
-					analyticsGroup="PluginsBrowser" />
-			);
-		}
+	getSearchBox() {
+		const { WrappedSearch } = this;
 
 		return (
-			<SearchCard
-				autoFocus={ ! hasTouch() }
+			<WrappedSearch
+				pinned
+				fitsContainer
 				onSearch={ this.doSearch }
 				initialValue={ this.props.search }
-				placeholder={ this.translate( 'Search Plugins' ) }
+				placeholder={ this.props.translate( 'Search Plugins' ) }
 				delaySearch={ true }
-				analyticsGroup="PluginsBrowser" />
+				analyticsGroup="PluginsBrowser"
+			/>
 		);
 	},
 
 	getNavigationBar() {
 		const site = this.props.site ? '/' + this.props.site : '';
-		return <SectionNav selectedText={ this.translate( 'Category', { context: 'Category of plugins to be filtered by' } ) }>
+		return <SectionNav selectedText={ this.props.translate( 'Category', { context: 'Category of plugins to be filtered by' } ) }>
 			<NavTabs label="Category">
 				<NavItem
 					path={ '/plugins/browse' + site }
 					selected={ false }
 				>
-					{ this.translate( 'All', { context: 'Filter all plugins' } ) }
+					{ this.props.translate( 'All', { context: 'Filter all plugins' } ) }
 				</NavItem>
 				<NavItem
 					path={ '/plugins/browse/featured' + site }
 					selected={ this.props.path === ( '/plugins/browse/featured' + site ) }
 				>
-					{ this.translate( 'Featured', { context: 'Filter featured plugins' } ) }
+					{ this.props.translate( 'Featured', { context: 'Filter featured plugins' } ) }
 				</NavItem>
 				<NavItem
 					path={ '/plugins/browse/popular' + site }
 					selected={ this.props.path === ( '/plugins/browse/popular' + site ) }
 				>
-					{ this.translate( 'Popular', { context: 'Filter popular plugins' } ) }
+					{ this.props.translate( 'Popular', { context: 'Filter popular plugins' } ) }
 				</NavItem>
 				<NavItem
 					path={ '/plugins/browse/new' + site }
 					selected={ this.props.path === ( '/plugins/browse/new' + site ) }
 				>
-					{ this.translate( 'New', { context: 'Filter new plugins' } ) }
+					{ this.props.translate( 'New', { context: 'Filter new plugins' } ) }
 				</NavItem>
 			</NavTabs>
-			{ this.getSearchBox( true ) }
+			{ this.getSearchBox() }
 		</SectionNav>;
 	},
 
-	getPageHeaderView() {
-		if ( this.props.category ) {
-			return this.getNavigationBar();
-		}
+	handleSuggestedSearch( term ) {
+		this.reinitializeSearch();
+		return () => this.doSearch( term );
+	},
+
+	getSearchBar() {
+		const suggestedSearches = [
+			this.props.translate( 'Engagement', { context: 'Plugins suggested search term' } ),
+			this.props.translate( 'Security', { context: 'Plugins suggested search term' } ),
+			this.props.translate( 'Appearance', { context: 'Plugins suggested search term' } ),
+			this.props.translate( 'Writing', { context: 'Plugins suggested search term' } ),
+		];
+
+		return (
+			<SectionNav
+				selectedText={ this.props.translate( 'Suggested Searches', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?